tidb.go

```diff
@@ -88,7 +89,7 @@ var (
 	}
 	stores = make(map[string]kv.Driver)
 	// Debug is the switch for pprof. If Debug is true, tidb will start with pprof on.
-	Debug = true
+	Debug = os.Getenv("TIDB_DEBUG") == "1"
```

Comment:
For `TIDB_PPROF`, should I create a new variable? 

If so, how does it affect `if` statement for starting listening?

Should it be `if Debug && EnablePprof` or just `if EnablePprof`?